lib/formfields/admin/domains/formfield.domains_edit.php

```diff
@@ -467,6 +489,13 @@
 						'value' => array(
 							$result['dkim']
 						)
+					),
+					'dkiminfo' => array(				// DF8OE
+						'visible' => (\Froxlor\Settings::Get('dkim.use_dkim') == '1' && $result['dkim'] && !$result['isbinddomain'] ? true : false),
+						'visible' => ($result['dkim'] && !$result['isbinddomain'] ? true : false),
+						'label' => 'add to DNS Zonefile:',
```

Comment:
missing translation

Reply:
Yes - I have not worked on database additions yet.

Reply:
database additions have nothing to do with the language files in `lng/`

Reply:
correct. I will add translation.